packages/react-ui-core/src/IncrementDecrementFilter/IncrementDecrementFilter.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component, PropTypes } from 'react'
+
+export default class IncrementDecrementFilter extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      count: this.props.count,
+    }
+    this.increment = this.increment.bind(this)
+    this.decrement = this.decrement.bind(this)
+  }
+  increment() {
+    const incrementValue = this.state.count + 1
+    this.setState({
+      count: incrementValue,
+    })
+  }
+  decrement() {
+    if (this.state.count > 1) {
+      const decrementValue = this.state.count - 1
+      this.setState({
+        count: decrementValue,
+      })
+    }
+  }
+  render() {
+    const { count } = this.state
+    const { theme, label, leftUnit, rightUnit, id } = this.props
+    return (
+      <div id={id} className={theme.container}>
+        <div className={theme.label}>
+          {label}
+        </div>
+        <div className={theme.componentContainer}>
+          <span
+            onClick={this.increment}
+            className={`leftUnit ${theme.contentInnards} ${theme.circle}`}
```

Comment:
i am not sure i like these classnames, we should not assume a style or flavor of button here 

for example `theme.circle` implies that this component will always be a circle

we want things generic as possible, i will explain more in the morning. 

also, please use the `classnames` library for concatting classes so if a theme  does not have it it is does not add a class of undefined.